src/cmd/sample.rs

```diff
@@ -36,6 +36,7 @@ Common options:
                            in the output.)
     -d, --delimiter <arg>  The field delimiter for reading CSV data.
                            Must be a single character. (default: ,)
+    --seed <number>        RNG seed.
```

Comment:
Since `--seed` is not a common option, this should be moved into a different section called `sample options:` that is above `Common options:`. See how it is done for other commands: https://github.com/BurntSushi/xsv/blob/master/src/cmd/fmt.rs#L19

Reply:
Done.